backend/src/bin/jobs/index.ts

```diff
@@ -12,9 +12,12 @@ import refreshSampleData from './refreshSampleData'
 import cleanUp from './cleanUp'
 import checkStuckIntegrationRuns from './checkStuckIntegrationRuns'
 import enrichOrganizations from './organizationEnricher'
+import { WEEKLY_EMAILS_CONFIG } from '@/conf'
+
+const EMAILS_ENABLED = WEEKLY_EMAILS_CONFIG.enabled === 'true'
 
 const jobs: CrowdJob[] = [
-  weeklyAnalyticsEmailsCoordinator,
+  EMAILS_ENABLED && weeklyAnalyticsEmailsCoordinator,
   integrationTicks,
```

Comment:
This will push a boolean to the array - can we check the env and push to the jobs array if true instead?